pkg/execute/executor.go

```diff
@@ -278,6 +278,12 @@ func runKubectlCommand(args []string, clusterName, defaultNamespace string, isAu
 		if strings.HasPrefix(arg, ClusterFlag.String()) {
 			// Check if flag value in current or next argument and compare with config.settings.clustername
 			if arg == ClusterFlag.String() {
+				nextArgasClusterName := args[index+1]
+				//This will remove 'http://' from clustername if it is automatically added
+				if strings.Contains(nextArgasClusterName, "<http://") {
+					nerCommands := strings.Split(nextArgasClusterName, "|")
+					args[index+1] = strings.Replace(nerCommands[1], ">", "", 1)
+				}
```

Comment:
@girishg4t we need the fix generic. Slack actually converts any string that matcher URL pattern into hyperlink. If we want to list CRDs/CRs e.g `@BotKube get prometheuses.monitoring.coreos.com`, in this case, the resource name will be converted into hyperlink. I had fixed this in one of the POCs where I had used regex to do that. Please take a look at this commit for reference: https://github.com/infracloudio/botkube/commit/08fe7731ddb698454cd7f680c4af2257627d266f

Reply:
Thanks @PrasadG193 i have corrected the code as per review comment, Please check and let me know if any change require